app/screens/home/StoreList.java

```diff
@@ -1,26 +1,68 @@
 package icebase.app.screens.home;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Scanner;
 import icebase.app.App;
+import icebase.app.api.API;
+import icebase.app.enums.SCREEN_ENUM;
 import icebase.app.MenuTitle;
+import icebase.app.Router;
 import icebase.app.screens.Screen;
+import icebase.app.types.Store;
+import icebase.icebase.Auth;
+import icebase.icebase.User;
 
 public class StoreList implements Screen {
     public void display() {
         Scanner sc = App.sc;
-
-        MenuTitle.displayMainTitle();
-        MenuTitle.displaySubTitle("[STORES]");
-        String[] storeList = { "Store 1", "Store 2" };
-        int range = storeList.length;
-        storeList[range - 1] = "Return";
         int choice;
+        Store chosenStore;
+        Auth currentUser = Auth.getAuth();
+
+        List<Store> stores = new ArrayList<>();
+        ArrayList<String> storeNames = new ArrayList<>();
+
+        // Gets names of each stores for options list
+        try {
+            stores = API.getStores();
+
+            for (Store store : stores) {
+                storeNames.add(store.getName());
+            }
+            storeNames.add("Return");
+
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+            return;
+        }
+
+        while (true) {
+            MenuTitle.displayMainTitle();
+            MenuTitle.displaySubTitle("STORES");
+            MenuTitle.displayOptions(storeNames);
+
+            try {
+                System.out.print("Choice: ");
+                choice = Integer.parseInt(sc.nextLine());
+                if (choice == storeNames.size()) {
+                    return;
+                }
+
+                chosenStore = stores.get(choice - 1);
+                if (chosenStore.getSellerId().equals(currentUser.getUser().getId())) {
```

Comment:
good idea nga ang seller id nalang ang i check instead of calling pa another API call, pero to clean up lang

extract the values into a variable para easy ma read ang code

```
String sellerId = chosenStore.getSellerId();

if  (sellerId.equals(currentUserId))
```